core/src/services/s3/backend.rs

```diff
@@ -302,7 +302,7 @@ pub struct S3Builder {
     enable_virtual_host_style: bool,
 
     http_client: Option<HttpClient>,
-    customed_credential_load: Option<Box<dyn AwsCredentialLoad>>,
+    custom_credential_load: Option<Box<dyn AwsCredentialLoad>>,
```

Comment:
It's part of our public API. Please don't change it.

Reply:
Unfortunately, this is a typo. Will do rollback first.